gateway/ln-gateway/src/lightning/ldk.rs

```diff
@@ -94,9 +131,29 @@ impl GatewayLdkClient {
             node_alias,
             ..Default::default()
         });
-        node_builder
-            .set_entropy_bip39_mnemonic(mnemonic, None)
-            .set_chain_source_esplora(esplora_server_url.to_string(), None);
+
+        node_builder.set_entropy_bip39_mnemonic(mnemonic, None);
+
+        let bitcoind_rpc = create_bitcoind(&chain_source_config.bitcoin_rpc_config())?;
```

Comment:
We probably shouldn't even call `create_bitcoind` if the chain source is esplora right?

Reply:
Oh I see it is general between the two. Nevermind :)

Reply:
Yeah, we should probably rename it. `create_bitcoind()` creates a generic bitcoin backend, not necessarily a bitcoind backend.